kclvm/tools/src/LSP/src/hover.rs

```diff
@@ -160,19 +161,28 @@ fn convert_doc_to_marked_string(doc: &(String, MarkedStringType)) -> MarkedStrin
 
 // Convert docs to Hover. This function will convert to
 // None, Scalar or Array according to the number of positions
-fn docs_to_hover(docs: Vec<(String, MarkedStringType)>) -> Option<lsp_types::Hover> {
-    match docs.len() {
+fn docs_to_hover(
+    docs: Vec<(String, MarkedStringType)>,
+    pkg_path: String,
```

Comment:
Why do you have to handle pkg_path separately, you can take pkg_path as MarkedString too